src/workbench/visualizers.ts

```diff
@@ -0,0 +1,22 @@
+import type {VisualizerExportModule} from '../visualizers/VisualizerInterface'
+
```

Comment:
This file is insufficiently DRY, i.e., there is too much repeated code with src/visualizers/visualizers.ts. Two options:
(1) You could have src/visualizers/visualizer.ts also glob the workbench directory if the environment variable is set, and just do all the work there.
(2) You could put the common code in a new file and just use that common code from both visualizers.ts

At my first glance, (1) is better; then there will be no need for an additional import and an additional line adding visualizers in Scope.vue; just the single import of VISUALIZERS from the standard visualizers.ts will get all of the desired ones depending on the environment variable.

Reply:
Excellent, thanks for reducing the code by an entire source file.